lib/compute.rb

```diff
@@ -0,0 +1,29 @@
+module Declarations
+  module prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
+
+        def initialize(declaration:, prefill:)
```

Comment:
The `initialize` method should validate the presence of `declaration` and `prefill` to ensure they are not `nil`.